sk0-back/src/auth/jwt.strategy.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { PassportStrategy } from '@nestjs/passport';
+import { Strategy, ExtractJwt } from 'passport-jwt';
+import { JwtPayload } from './jwt-payload.interface';
+import { UsersService } from 'src/users/services/users.service';
+
+@Injectable()
+export class JwtStrategy extends PassportStrategy(Strategy) {
+  userService: UsersService;
+  constructor() {
+    super({
+      jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+      secretOrKey: process.env.JWT_SECRET,
```

Comment:
Not sure why you are using the built-in JWT module, but overwriting it with a custom JWTStrategy. 🤔 The config should used here too.

Reply:
that might be the problem why it doesn't compile :D